cylc/uiserver/scripts/hub.py

```diff
@@ -24,11 +24,13 @@
 
 from cylc.uiserver import (
     __version__,
-    __file__ as uis_pkg
+    __file__ as uis_pkg,
+    init_log
 )
 
 
 def main(*args):
+    init_log()
```

Comment:
I don't think we need to call this here, the `hub` command doesn't start a UIServer?

Reply:
Removed, thanks!